src/pages/Welcome.jsx

```diff
@@ -4,15 +4,15 @@ import Glasses from "../pictures/WinesColor.png";
 
 function Welcome() {
   return (
-    <div className="box">
+    <main className="box">
       <h1 className="title">Bienvenue</h1>
       <div className="glasses">
-        <img src={Glasses} alt="glasses of wine" />
+        <img src={Glasses} alt="" />
```

Comment:
c'est une image purement décorative : mieux vaut un alt vide :)

https://html.spec.whatwg.org/#a-purely-decorative-image-that-doesn't-add-any-information